ding/policy/sac.py

```diff
@@ -713,10 +713,10 @@ def _init_learn(self) -> None:
 
         # Weight Init for the last output layer
         init_w = self._cfg.learn.init_w
-        self._model.actor[2].mu.weight.data.uniform_(-init_w, init_w)
-        self._model.actor[2].mu.bias.data.uniform_(-init_w, init_w)
-        self._model.actor[2].log_sigma_layer.weight.data.uniform_(-init_w, init_w)
-        self._model.actor[2].log_sigma_layer.bias.data.uniform_(-init_w, init_w)
+        self._model.actor[-1].mu.weight.data.uniform_(-init_w, init_w)
+        self._model.actor[-1].mu.bias.data.uniform_(-init_w, init_w)
+        self._model.actor[-1].log_sigma_layer.weight.data.uniform_(-init_w, init_w)
+        self._model.actor[-1].log_sigma_layer.bias.data.uniform_(-init_w, init_w)
```

Comment:
why -1?

Reply:
because the original nn framework is _Linear, activation, head_, the actor model has index [0, 1, 2].
now we change that part to encoder(which including activation) and head, so the actor model just has index [0, 1].
To accommodate both situations, change the _2_ to _-1_.